src/apis/fetch.ts

```diff
@@ -8,7 +8,7 @@ export const fetch = (token: Function): Function => (
 ): Promise<Response> => {
   const { method, headers, raw, ...rest } = options
 
-  if (body && !(body instanceof Blob)) {
+  if (body && !(Blob && body instanceof Blob)) {
```

Comment:
Why is this file in this PR? 🤔

Reply:
This is used to remove the crash when trying to export in NodeJS (See #9). I guess we can say it's about "developer experience", so it could be inserted in this PR 😬